python/tests/test_highlevel.py

```diff
@@ -2448,6 +2448,114 @@ def test_tree_node_edges(self):
                         assert edge.right >= tree.interval.right
             assert np.all(edge_visited)
 
+    def verify_individual_vectors(self, ts):
+        verify_times = np.repeat(np.nan, ts.num_individuals)
+        verify_populations = np.repeat(tskit.NULL, ts.num_individuals)
+        for ind in ts.individuals():
+            if len(ind.nodes) > 0:
+                t = {ts.node(n).time for n in ind.nodes}
+                p = {ts.node(n).population for n in ind.nodes}
+                assert len(t) <= 1
+                assert len(p) <= 1
+                verify_times[ind.id] = t.pop()
+                verify_populations[ind.id] = p.pop()
+
+        times = ts.individuals_time
+        populations = ts.individuals_population
+        assert np.array_equal(times, verify_times, equal_nan=True)
+        assert np.array_equal(populations, verify_populations, equal_nan=True)
+        times2 = ts.individuals_time
+        populations2 = ts.individuals_population
+        assert np.array_equal(times, times2, equal_nan=True)
+        assert np.array_equal(populations, populations2, equal_nan=True)
+        # check aliases also
+        times3 = ts.individuals_time
+        populations3 = ts.individuals_population
+        assert np.array_equal(times, times3, equal_nan=True)
+        assert np.array_equal(populations, populations3, equal_nan=True)
+
+    def test_individuals_population_errors(self):
+        t = tskit.TableCollection(sequence_length=1)
+        t.individuals.add_row()
+        t.individuals.add_row()
+        for j in range(2):
+            t.populations.add_row()
+            t.nodes.add_row(time=0, population=j, individual=0)
+        ts = t.tree_sequence()
+        with pytest.raises(_tskit.LibraryError, match="population"):
```

Comment:
Nitpicky - but I'm trying to move to a standard where we use the macro for the string match. e.g: 
```suggestion
        with pytest.raises(_tskit.LibraryError, match="TSK_ERR_INDIVIDUAL_POPULATION_MISMATCH"):
```